hedyorg/hedy
# [FIX] Load more resources from the CDN

After looking at the logs, the top resources that still weren't being loaded from the CDN were:

```
/vendor/skulpt-stdlib-extensions.js
/images/Hedylightbulb_1.svg
/images/hero-graphic/hero-graphic.png
/images/hero-graphic/hero-graphic-nl.png
```

Use the `{{static()}}` function to force using the URL that will be routed to the CDN in production.

For the `skulpt-stdlib-extensions.js` file, this is bit more effort because the source of that URL originates in JavaScript, so we pass the static URL to JavaScript and have it do the concatenation.

**How to test**

Can only be tested after being deployed.

Open the Network Inspector. Run a program. Observe that `/vendor/skulpt-stdlib-extensions.js` is loaded from the CDN.
